tools/ktx/command.h

```diff
@@ -289,23 +294,34 @@ struct OptionsMultiInSingleOut {
 
     void init(cxxopts::Options& opts) {
         opts.add_options()
+                ("stdin", "Use stdin as the input file. (Using a single dash '-' as the input file has the same effect)")
+                ("stdout", "Use stdout as the output file. (Using a single dash '-' as the output file has the same effect)")
                 ("files", "Input/output files. Last file specified will be used as output", cxxopts::value<std::vector<std::string>>(), "<filepath>");
         opts.parse_positional("files");
         opts.positional_help("<input-file...> <output-file>");
     }
 
     void process(cxxopts::Options&, cxxopts::ParseResult& args, Reporter& report) {
         std::vector<std::string> files;
-        if (args.count("files"))
-            files = args["files"].as<std::vector<std::string>>();
+        if (args.count("stdin"))
+            files.emplace_back("-");
+        if (args.count("stdout"))
+            files.emplace_back("-");
+        if (args.count("files")) {
+            const auto& argFiles = args["files"].as<std::vector<std::string>>();
+            files.insert(files.end(), argFiles.begin(), argFiles.end());
+        }
         if (files.size() < 1)
             report.fatal_usage("Input and output files must be specified.");
         if (files.size() < 2)
-            report.fatal_usage("Output file must be specified.");
+            report.fatal_usage("{} file must be specified.", args.count("stdout") == 0 ? "Output" : "Input");
 
         outputFilepath = std::move(files.back());
         files.pop_back();
         inputFilepaths = std::move(files);
+
+        if (std::count(inputFilepaths.begin(), inputFilepaths.end(), "-") > 1)
+            report.fatal_usage("'-' or --stdin as input file was specified more than once.");
```

Comment:
Is the CLI syntax to write `--stdin` in the input file list where you want stdin used, e.g. `a --stdin c`?

Reply:
`--stdin` will prepend a single `-` to the input file list so if there are multiple input files than the first file will be read from the stdin.
`-` is supported both as input and output for aliasing stdin and stdout and its position can vary freely in the input/output list

Reply:
That wasn't entirely clear to me from the option help lines that I read while reviewing. Please improve the option help lines in `command.h`. (The line here is fine. It is just where I realized the possible confusion.)